test/parallel/test-url-format.js

```diff
@@ -253,6 +253,13 @@ const formatTests = {
     auth: '\uD83D\uDE00',
     hostname: 'www.example.com',
     pathname: '/'
+  },
+  // ipv6
```

Comment:
There is already an IPv6 section, please append the test there.

Reply:
Thanks, the code is updated